dist/appConfig.js

```diff
@@ -73,6 +73,8 @@ configuration.queryparams =
         'SSBatchProcessorStatusMessages': '/status/',
         'SSBatchProcessorStreamGrids': '/streamgrids/',
         'SSBatchProcessorSubmitBatch': '/batch',
+        'SSBatchProcessorRefreshBatch': '/batch/{0}/refresh?deleteCurrentData={1}',
+        'SSBatchProcessorStartWorker': '/worker',

```

Comment:
Do we need to add these to the other appconfigs?

Reply:
Yes, woops. Totally forgot about that!